plugins/jfrog-artifactory/src/components/JfrogArtifactoryRepository/tableHeading.tsx

```diff
@@ -29,6 +34,36 @@ export const columns: TableColumn[] = [
     field: 'size',
     type: 'string',
   },
+  {
+    title: 'Vulnerabilities (Critical)',
+    field: 'vulnerabilities_critical',
+    type: 'numeric',
+  },
+  {
+    title: 'Vulnerabilities (High)',
+    field: 'vulnerabilities_high',
+    type: 'numeric',
+  },
+  {
+    title: 'Vulnerabilities (Medium)',
+    field: 'vulnerabilities_medium',
+    type: 'numeric',
+  },
+  {
+    title: 'Vulnerabilities (Low)',
+    field: 'vulnerabilities_low',
+    type: 'numeric',
+  },
+  {
+    title: 'Vulnerabilities (Info)',
+    field: 'vulnerabilities_info',
+    type: 'numeric',
+  },
```

Comment:
Can we merge this into a single field? Similar to what we have in Quay plugin? I think it would be nice if we can make the experience consistent across plugins. 🙂 

Take a look here: https://github.com/janus-idp/backstage-plugins/tree/main/plugins/quay#procedure

https://github.com/janus-idp/backstage-plugins/blob/2ca2d7afe18a4f5d858e5a43c908228ef828792e/plugins/quay/src/components/QuayRepository/tableHeading.tsx#L9

Reply:
Hey @tumido 

Absolutely this would be much nicer indeed. 👍